inc/mailcollector.class.php

```diff
@@ -977,8 +973,10 @@ function buildTicket($uid, \Laminas\Mail\Storage\Message $message, $options = []
 
       // manage blacklist
       $blacklisted_emails   = Blacklist::getEmails();
-      // Add name of the mailcollector as blacklisted
-      $blacklisted_emails[] = $this->fields['name'];
+      if (GLPIMailer::ValidateAddress($this->fields['name'])) {
```

Comment:
should not it be blacklisted if not valid?

Reply:
As it is just a comparison with from, we indeed do not care if it is a valid email or not.